src/components/CardSchemes/components/PaymentMethodIcon/PaymentMethodIcon.spec.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { keys, values } from 'lodash/fp';
+import PaymentMethodIcon from './PaymentMethodIcon';
+// eslint-disable-next-line max-len
+import schemeMap, {
+  iconComponents
+} from '../../../CreditCardDetails/components/scheme-icons/card-scheme-icons';
+
+describe('PaymentMethodIcon', () => {
+  /**
+   * Style tests.
+   */
+  it('should render with default styles', () => {
+    const actual = create(
+      <div>
+        {keys(schemeMap).map(schemeId => (
+          <PaymentMethodIcon
```

Comment:
There is no need to render multiple `PaymentMethodIcon`s in this test. You are testing the styles of a single component.